internal/app/machined/pkg/runtime/logging/zap.go

```diff
@@ -0,0 +1,142 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I think we can move this several levels up to `pkg/logging`, as these functions might be useful for other components as well? 

they seem to be pretty generic and really not specific to machined/runtime